radixdlt/src/test/java/com/radixdlt/network/transport/udp/UDPTransportOutboundConnectionTest.java

```diff
@@ -81,6 +81,7 @@ public UDPTransportOutboundConnectionTest(String sourceAddressStr, String destin
 
 	@Before
 	public void setUp() throws UnknownHostException {
+	    Thread.interrupted();
```

Comment:
What does this serve?
Also, indent looks a bit weird

Reply:
Indentation fixed. The interrupted flag gets cleared after calling the Thread.interrupted() method, and so will not be propagated to the test which is just about to be executed.